source/_docs/guides/github-pull-requests.md

```diff
@@ -0,0 +1,340 @@
+  <div style="margin-bottom:30px;" class="col-md-4">
+  <div class="plugin-dir">
+      <div class="pantheon-official">
+        <div class="main-topic-info__plugin-image" style="background-image:url(/source/docs/assets/images/composer-logo.svg)"></div>
+        <p>Composer</p>
+      </div>
+      <div class="terminus-plugin">
+          <h3 class="plugin-title">Composer</h3>
+          <p class="topic-info__description"><a href="https://getcomposer.org">Composer</a> is a PHP dependency manager that provides an alternative, more modern way to manage the external code used by a project. For example, Composer may be used to install the modules and themes used by a Drupal site.</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+*Pull requests* are a formalized way of reviewing and merging a proposed set of changes to a codebase. When one member of a development team makes changes to a project, all of the files modified to produce the feature are committed to a separate branch, and that branch becomes the basis for the pull request. GitHub allows other team members to review all of the differences between the new files and their original versions, before *merging* the pull request.
+
+In the workflow set up in this guide, a multidev environment is created for each pull request branch. Work in these environments can also be committed back to the same branch for review on GitHub. When done, the result is merged into the dev environment.
+
+![Multidev PR workflow](/source/docs/assets/images/pr-workflow/multidev-git-pr-workflow.png)
+
+It is also common to set up automated tests to confirm that the project is working as expected; when tests are available, GitHub will run them and display the results of the tests with the pull request. Working on projects with comprehensive tests increases the development team's confidence that submitted pull requests will work correctly when they are integrated into the main build.
+
+When using GitHub and Composer to manage a Drupal site, only those files unique to the project are part of the project's main repository. Composer is used to fetch the external code needed by the project; a process running on Circle CI executes Composer, and ensures that the final composed build results are installed on Pantheon:
+
+![Artifact Deployment](/source/docs/assets/images/artifact-deployment.png)
+
+One advantage of managing code this way is that it keeps the change sets (differences) for pull requests as small as possible. If a pull request upgrades several external projects, only the external dependency metadata file will change; the actual code changes in the upgraded projects themselves are not shown.
+
+Generally, use of Composer is optional; however, some Drupal modules, such as the Address module require the use of Composer. If a site needs just one module that requires Composer, then it should manage all of its modules with Composer.
+
+## Before You Begin
+
+1.  To prepare your system for local development, install:
+
```

Comment:
Minor nit: there's more whitespace above the list than below, which looks odd. Not necessary to fix this.